### PR TITLE
Quick fix: Gottschalk reconstruction

### LINKED ISSUE #807: Show a placeholder page in the manuscript detail view when there are lacunae

### LINKED ISSUE #813: Improve `GOTTSCHALK` reconstruction view
Images are currently of different rotations and sizes...see what can be done (for now, on an ad-hoc basis) to fix this...

### LINKED ISSUE #814: GOTTSCHALK mapping issues?
47r shows nothing at all: 
image

57r fails to load chants: 
image

If you scroll back up, the images change but it still says 57r on the right for a while. 

52v sometimes fails to load chants:
image

### PR DESCRIPTION
**Pull request by dchiller**
This PR introduces an interim approach for serving a usable Gottschalk manuscript reconstruction on CU. In particular, it serve certain images of the manuscript and an IIIF manifest for the reconstruction on CU itself, allowing CU to work from edited versions of the manifest and images. Those edits include:

- the addition of lacunae in the manifest (an image representing a lacuna is also added)
- resizing of certain images that were well outside the norm of dimensions for images in the reconstruction
- rotation of certain images so the top of the folio is at the top of the image

Closes #814 Closes #813 Closes #807 

Anticipating that we will merge this PR, I've opened #822 to track the development of a longer-term, sustainable fix.

### CODE REVIEW

File: cantaloupe/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+ENV JAVA_HOME=/opt/java/openjdk
+ENV PATH="${JAVA_HOME}/bin:${PATH}"
+
+# Install various dependencies:
+RUN apt-get update && apt-get install -y \
+    unzip \
+    wget
+
+# Download OpenJPEG binaries and unpack them:
```
Comment (dchiller) [inline cantaloupe/Dockerfile:11]: Document reason for OpenJPEG

Comment (dchiller) [reply cantaloupe/Dockerfile:11]: explain that it's a dependency of cantaloupe for serving jpeg files...

Comment (dchiller) [reply cantaloupe/Dockerfile:11]: why downloading wget

File: cantaloupe/cantaloupe.properties
Comment (dchiller) [inline cantaloupe/cantaloupe.properties:1]: simplify this?

**Review by jacobdgm — APPROVED** [review]
Having walked through this with you, @dchiller, I'm confident that everything is here for a reason and makes sense. Would suggest tweaking the couple of things we made note of, but in general, this looks good.

**Pull request merged**